src/haddock/modules/analysis/seletopclusts/__init__.py

```diff
@@ -26,6 +26,15 @@ def confirm_installation(cls):
 
     def _run(self):
         """Execute the module's protocol."""
+
+        if self.params["top_models"] <= 0:
```

Comment:
Okay, fine for now but I need to find a better way around this. Also to make sure the right types are passed in the user config files.